src/dist/pkg.rs

```diff
@@ -169,8 +169,20 @@ mod toolchain_imp {
                 builder.append_dir(tar_path, dir_path)?
             }
             for (tar_path, file_path) in file_set.into_iter() {
-                let file = &mut fs::File::open(file_path)?;
-                builder.append_file(tar_path, file)?
+                let file = &mut fs::File::open(&file_path)?;
+                builder.append_file(&tar_path, file)?;
+                // Should have no need to package clang-cl.exe since the dist
+                // server must a Linux environemnt.
```

Comment:
```suggestion
                // Should have no need to package clang-cl.exe since the dist
                // server must a Linux environment.
```
I don't quite understand this comment: isn't `clang-cl` only used on Windows, therefore we want `clang-cl.exe`? Or, is `clang-cl` (without `.exe`) used on other platforms for cross-compiling? What is the "dist server" in this context?